Update create-a-kubernetes-blue-green-deployment.md (#2307)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/create-a-kubernetes-blue-green-deployment.md

Click the **Stage Deployment** step. The step simply includes a name, timeout, and Skip Dry Run options.

> **Skip Dry Run:** By default, Harness uses the `--dry-run` flag on the `kubectl apply` command during the **Initialize** step of this command, which prints the object that would be sent to the cluster without really sending it. If the **Skip Dry Run** option is selected, Harness will not use the `--dry-run` flag. The first time you deploy, the **Stage Deployment** step creates two Kubernetes services, a new pod set, and deploys your app to the pod set.

Reviewer: ```suggestion
**Skip Dry Run:** By default, Harness uses the `--dry-run` flag on the `kubectl apply` command during the **Initialize** step of this command, which prints the object that would be sent to the cluster without actually sending it. If the **Skip Dry Run** option is selected, Harness will not use the `--dry-run` flag. The first time you deploy, the **Stage Deployment** step creates two Kubernetes services, a new pod set, and deploys your app to the pod set.
```

> **Skip Deployment if Using the Manifest Used in a Previous Deployment**: This functionality is currently beta and behind the feature flag, `CDS_SUPPORT_SKIPPING_BG_DEPLOYMENT_NG`. When running the **Stage Deployment** step, if you select this option, Harness render the manifests, and compare the manifests with the last deployed manifests to see if there are any changes. If there are no changes in the manifests used in the step and the previous deployment, Harness skips the step and progresses to the subsequent steps in the pipeline. This ensures that no routes or labels associated with the primary or stage (Blue or Green) are manipulated when there are no manifest changes present. 

Reviewer: ```suggestion
**Skip Deployment if Using the Manifest Used in a Previous Deployment**: This functionality is currently in beta and behind the feature flag, `CDS_SUPPORT_SKIPPING_BG_DEPLOYMENT_NG`. When running the **Stage Deployment** step, if you select this option, Harness renders the manifests and compares them with the last deployed manifests to see if there are any changes. If there are no changes in the manifests used in the step and the previous deployment, Harness skips the step and progresses to the subsequent steps in the pipeline. This ensures that no routes or labels associated with the primary or stage (Blue or Green) are manipulated when no manifest changes are present. 
```

> **Skip Deployment if Using the Manifest Used in a Previous Deployment**: This functionality is currently beta and behind the feature flag, `CDS_SUPPORT_SKIPPING_BG_DEPLOYMENT_NG`. When running the **Stage Deployment** step, if you select this option, Harness render the manifests, and compare the manifests with the last deployed manifests to see if there are any changes. If there are no changes with the manifests used in the step and the previous deployment, Harness skips the step and progresses to the subsequent steps in the pipeline. This ensures that no routes or labels associated with the primary or stage (Blue or Green) are manipulated when there are no manifest changes present. 

Reviewer 2: ```suggestion
**Skip Deployment if Using the Manifest Used in a Previous Deployment**: This functionality is currently beta and behind the feature flag, `CDS_SUPPORT_SKIPPING_BG_DEPLOYMENT_NG`. When running the **Stage Deployment** step, if you select this option, Harness render the manifests, and compare the manifests with the last deployed manifests to see if there are any changes. If there are no changes in the manifests used in the step and the previous deployment, Harness skips the step and progresses to the subsequent steps in the pipeline. This ensures that no routes or labels associated with the primary or stage (Blue or Green) are manipulated when there are no manifest changes present. 
```
